themes/jb/layouts/partials/meta.html

```diff
@@ -1,9 +1,9 @@
 <!-- Meta Ttitles -->
-<title itemprop="name">{{ .Title -}}{{- if .Title }} | {{end}}{{ .Site.Title }}</title>
-<meta property="og:title" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ .Site.Title }}" />
-<meta name="twitter:title" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ .Site.Title }}" />
-<meta itemprop="name" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ .Site.Title }}" />
-<meta name="application-name" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ .Site.Title }}" />
+<title itemprop="name">{{ .Title -}}{{- if .Title }} | {{end}}{{ if ne .Title (print .CurrentSection.Title ` ` .Params.episode) }}{{ .Params.show_name -}}{{- if .Params.show_name }} {{ if .Params.episode }}{{ .Params.episode -}}{{end}} | {{end}}{{end }}{{ .Site.Title }}</title>
```

Comment:
So this is more preference than anything, but I think leaving these Hugo blocks in a long one-liner is *very* difficult to read. I would love to see it broken up into separate lines.

Reply:
I can give it a shot this evening! The challenge I had was making sure I didn’t inadvertently linebreak a field once rendered. I’ll play with the template linebreak handler logic and report back.

Reply:
I could be wrong, but I thought it was similar to a `p` tag where it would not insert a newline unless you added a line break tag (`<br/>`) or code ( I think either  `&#013;` or `&#010;`? Don't quote me there).

Reply:
@elreydetoda @CGBassPlayer 

Roger that. I think I understand that what's rendered won't linebreak unless we have an explicit linebreak tag. For the `<title>` tag, I imagine it shouldn't be a functional problem; it would just look funny to have the source broken up. In the `<meta>` tags that follow, though, I wasn't as sure because the value we're modifying are in attributes instead of contained between tags. I don't do a lot of web dev, so I'm not sure how that plays out.

All that said, I think I've got this reorganized in a succinct way. PTAL when you've got a moment!